usaspending_api/api_contracts/contracts/v2/disaster/recipient/type/spending.md

```diff
@@ -0,0 +1,161 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Type Disaster/Emergency Funding [/api/v2/disaster/recipient/type/spending/]
```

Comment:
I'm iffy on using `recipient/type` here instead of just `recipient_type`. Obviously recipients and recipient types have a business connection, but at the same time "type" isn't really  a property of what you would get from the `recipient` endpoint; they are essentially different dimensions on the data as far as I can tell.

I doubt this will lead to any major misunderstandings, just a style compliant.

Reply:
May be a valid reason to change "type" to "busniess_category". 
or make type mores specific to `type_of_business`.

e..g `/api/v2/disaster/recipient/business_category/spending/`

Reply:
Good idea. Done.